project/Dependencies.scala

```diff
@@ -2,116 +2,189 @@ import sbt._
 
 object Dependencies {
 
+  private val config = Seq("com.typesafe" % "config" % "1.3.3")
+
+  private val pureConfig = Seq("com.github.pureconfig" %% "pureconfig" % "0.10.2")
+
+  private val scopt = Seq("com.github.scopt" %% "scopt" % "4.0.0-RC2")
+
+  private val logging = Seq(
+    "ch.qos.logback"             % "logback-classic"          % "1.2.3",
+    "net.logstash.logback"       % "logstash-logback-encoder" % "5.3",
+    "com.typesafe.scala-logging" %% "scala-logging"           % "3.7.2"
+  )
+
   private val akkaVersion = "2.5.21"
```

Comment:
I personally preferred all external versions to be collected together, instead of spread near to their actual dependency list.
That's because usually you want to easily scan the currently used versions of those libraries, and now it's not so easy.

Some use the convention to put the versions inside a local `Version` object, to keep 'em bundled, and use them as `Version.akka` for example.

Reply:
Yea, that will serve the purpose better I guess. Just moved versions to `Versions` object and lets see.